app/src/organisms/ProtocolDetails/RobotConfigurationDetails.tsx

```diff
@@ -138,41 +145,48 @@ export const RobotConfigurationDetails = (
         </>
       ) : null}
-      {requiredModuleDetails != null
-        ? requiredModuleDetails.map((module, index) => {
-            return (
-              <React.Fragment key={index}>
-                <Divider marginY={SPACING.spacing12} width="100%" />
-                <RobotConfigurationDetailsItem
-                  label={t('run_details:module_slot_number', {
-                    slot_number:
-                      getModuleType(module.params.model) ===
-                      THERMOCYCLER_MODULE_TYPE
-                        ? getSlotsForThermocycler(robotType)
-                        : module.params.location.slotName,
-                  })}
-                  item={
-                    <>
-                      <ModuleIcon
-                        key={index}
-                        moduleType={getModuleType(module.params.model)}
-                        marginRight={SPACING.spacing4}
-                        alignSelf={ALIGN_CENTER}
-                        color={COLORS.darkGreyEnabled}
-                        height={SIZE_1}
-                        minWidth={SIZE_1}
-                        minHeight={SIZE_1}
-                      />
-                      <StyledText as="p">
-                        {getModuleDisplayName(module.params.model)}
-                      </StyledText>
-                    </>
-                  }
-                />
-              </React.Fragment>
-            )
-          })
-        : null}
+      {requiredModuleDetails.map((module, index) => {
+        return (
+          <React.Fragment key={`module_${index}`}>
+            <Divider marginY={SPACING.spacing12} width="100%" />
+            <RobotConfigurationDetailsItem
+              label={
+                getModuleType(module.params.model) === THERMOCYCLER_MODULE_TYPE
+                  ? getSlotsForThermocycler(robotType)
+                  : module.params.location.slotName
+              }
+              item={
+                <>
+                  <ModuleIcon
+                    key={index}
+                    moduleType={getModuleType(module.params.model)}
+                    marginRight={SPACING.spacing4}
+                    alignSelf={ALIGN_CENTER}
+                    color={COLORS.darkGreyEnabled}
+                    height={SIZE_1}
+                    minWidth={SIZE_1}
+                    minHeight={SIZE_1}
+                  />
+                  <StyledText as="p">
+                    {getModuleDisplayName(module.params.model)}
+                  </StyledText>
+                </>
+              }
+            />
+          </React.Fragment>
+        )
+      })}
+      {requiredFixtureDetails.map((fixture, index) => {
+        return (
+          <React.Fragment key={`fixture_${index}`}>
+            <Divider marginY={SPACING.spacing12} width="100%" />
+            <RobotConfigurationDetailsItem
+              label={fixture.params.location.cutout}
+              item={getFixtureDisplayName(fixture.params.loadName)}
```

Comment:
```suggestion
              item={   <StyledText as="p">
 getFixtureDisplayName(fixture.params.loadName)  </StyledText> }
```
<img width="463" alt="Screen Shot 2023-10-02 at 12 11 37 PM" src="https://github.com/Opentrons/opentrons/assets/66035149/bfa99b6b-e3cf-4d27-8590-6f1f0bfef5e2">